cumulusci/robotframework/pageobjects/BasePageObjects.py

```diff
@@ -221,44 +222,18 @@ def select_dropdown_value(self, label, value):
         If a modal is present, the modal will be searched for the dropdown. Otherwise
         the first matching dropdown on the entire web page will be used.
         """
-        input_element = self._get_form_element_for_label(label)
-        if input_element is None:
-            raise Exception(f"No form element found with the label '{label}'")
-
-        # SeleniumLibrary's scroll_element_into_view doesn't seem to work
-        # for elements in a scrollable div on Firefox. Javascript seems to
-        # work though.
-        self.selenium.driver.execute_script(
-            "arguments[0].scrollIntoView()", input_element
-        )
-
-        if input_element.get_attribute("role") == "combobox":
-            # new lightning combobox. So much easier!
-            self.builtin.log(f"New style combo for {label}", "DEBUG")
-            input_element.click()
-            try:
-                item = input_element.find_element_by_xpath(
-                    f'//lightning-base-combobox-item[.="{value}"]'
-                )
-                item.click()
-            except NoSuchElementException:
-                raise Exception(f"Dropdown value '{value}' not found")
-        else:
-            self.builtin.log(f"Old style combo for {label}", "DEBUG")
-            trigger = input_element.find_element_by_class_name("uiPopupTrigger")
-            trigger.click()
-
-            try:
-                popup_locator = "//div[contains(@class, 'uiPopupTarget')][contains(@class, 'visible')]"
-                popup = self.selenium.get_webelement(popup_locator)
-            except ElementNotFound:
-                raise ElementNotFound("Timed out waiting for the dropdown menu")
+        locator = f"label:{label}"
+        try:
+            element = self.selenium.get_webelement(locator)
+        except Exception:
```

Comment:
Is there a way to be more specific with the exceptions type here?

Reply:
yeah, you're right. I was being lazy. Thanks for the suggestion.